agentassist/src/app/components/header/header.component.ts

```diff
@@ -285,8 +287,9 @@ export class HeaderComponent implements OnInit, OnDestroy {
             groups: [],
             users: usersList
           }
-        }
-        this.service.invoke('put.sharebot', params, payload, {headers: {'agentassist': true}}).subscribe(
+        };
+        const headers = new HttpHeaders().set('agentassist' , 'true').set('x-http-method-override','put');
```

Comment:
'x-http-method-override' header is already been added in service-invoker.service.ts. Please remove it here.

Why do we need 'agentassist' as an header?

Reply:
added as said by backend developer. agentassist:true is added as said by Venkatesh

Reply:
Approving the change for now. I think we need "agentassist" header for all the api calls, so this should be added in the generic file.